lib/stencil-push.utils.js

```diff
@@ -0,0 +1,329 @@
+'use strict';
+const _ = require('lodash');
+const async = require('async');
+const Bundle = require('./stencil-bundle');
+const fs = require('fs');
+const Inquirer = require('inquirer');
+const os = require('os');
+const ProgressBar = require('progress');
+const themeApiClient = require('./theme-api-client');
+const themePath = process.cwd();
+const themeConfig = require('./theme-config').getInstance(themePath);
+const uuid = require('uuid4');
+const utils = {};
+const Wreck = require('wreck');
+
+const bar = new ProgressBar('Uploading theme [:bar] :rate/bps :percent :etas', {
+    complete: '=',
+    incomplete: ' ',
+    total: 100,
+});
+
+module.exports = utils;
+
+function validateOptions(options, fields) {
+    options = options || {};
+    fields = fields || [];
+
+    fields.forEach(field => {
+        if (!_.has(options, field)) {
```

Comment:
oh, this will allow nested fields, like so:

`options = validateOptions(options, ['config.normalStoreUrl']);`